### PR TITLE
Get rid of pandas module usage

### PR DESCRIPTION
**Pull request by julmon**
By implementing our own pivot table function.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     packages=find_packages(),
     scripts=['temboard'],
     install_requires=[
-        'pandas>=0.15.0',
+        'dateutil>=1.5',
```
Comment (bersace) [inline setup.py]:
```
  Could not find a version that satisfies the requirement dateutil>=1.5 (from temboard==1.0a2.dev52->-r requirements-ci.txt (line 9)) (from versions: )

No matching distribution found for dateutil>=1.5 (from temboard==1.0a2.dev52->-r requirements-ci.txt (line 9))
```

File: temboardui/plugins/monitoring/pivot.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import operator
+
+
+def get_csv_data(fd):
+    # CSV parser as generator
+    for r in csv.DictReader(fd):
+        yield r
+
+
+def pivot_timeserie(fd, index, key, value, output):
```
Comment (bersace) [inline temboardui/plugins/monitoring/pivot.py]: Example TU:

``` python
# tests/unit/test_pivot.py
def test_pivot():
    from … import pivot_timeserie

    in_ = StringIO("…")
    out = StringIO(
    out = pivot_timeserie(in_, index='date', key='spcname', value='size', output=out_)
    data = out.getvalue()

    assert …
```

**Comment by julmon** [comment]
Just made some really basics benchmark tests using a CSV data file with 749 rows. Average runtime (10000 executions) are : 
- with `pandas` : **0.00295** s
- with `pivot_timeserie(): **0.00339** s

File: temboardui/plugins/monitoring/pivot.py (hunk 2 above)
Comment (julmon) [reply temboardui/plugins/monitoring/pivot.py]: Done. Thank you.

**Pull request merged**